warmup/summary_statistics.py

```diff
@@ -124,12 +124,7 @@ def collect_summary_statistics(data_dictionaries, half_bound, delta, steady_stat
                     else:
                         error_iter = (error_iter,)
                     median_time_to_steady, error_time_to_steady = bootstrap_confidence_interval(time_to_steadys)
-                    # Check whether the error is wider than the median, OR
-                    # whether the lower CI of the time to reach a steady state
-                    # implies that one or more of the early iterations took
-                    # < 0 seconds to complete.
-                    if (error_time_to_steady > median_time_to_steady or
-                        (median_time_to_steady - error_time_to_steady) / median_iter < 1.0):
+                    if error_time_to_steady > median_time_to_steady:
```

Comment:
I tend to agree with that: can we have `>` for steady iter (#) and `>=` for steady iter (s) easily enough?